src/components/gameAnswers/gameAnswers.js

```diff
@@ -0,0 +1,42 @@
+import './gameAnswers.css';
+import isAnswerCorrect from '../../services/game/isAnswerCorrect';
+
+export function gameAnswers(answers, rightAnswer, callback) {
```

Comment:
callback is not clear name either, name should say what exactly it does so if callback takes all possible answers and correct one -> and in result it validate if the player answer was correct the name of this callback shoudl be something like `correctAnswerValidator`